llvm/lib/CodeGen/InlineSpiller.cpp

```diff
@@ -1071,8 +1071,7 @@ void InlineSpiller::insertReload(Register NewVReg,
 static bool isRealSpill(const MachineInstr &Def) {
   if (!Def.isImplicitDef())
     return true;
-  assert(Def.getNumOperands() == 1 &&
-         "Implicit def with more than one definition");
+
```

Comment:
If we really want to keep an assert here, we could check that we have only one explicit operand.
Don't think that's worth it though.

Reply:
The verifier handles that case